src/LocalDatastore.js

```diff
@@ -41,29 +40,30 @@ const LocalDatastore = {
     controller.clear();
   },
 
-  _handlePinWithName(name: string, object: ParseObject) {
+  _handlePinWithName(name: string, object: any) {
```

Comment:
Why loose the typing?

Reply:
I kept running into a 
```
Uncaught TypeError: Super expression must either be null or a function, not undefined
```
Whenever I used ParseObject in the tests. I found a fix for it so I'll add the typing back.

Reply:
You are 💯